src/screens/DonateScreen/SegmentedControl.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from "react";
+import { PixelRatio, StyleSheet, View } from "react-native";
+import Text from "../../components/Text";
+import Touchable from "../../components/Touchable";
+import {
+  inputFieldBorderColor,
+  eucalyptusGreenColor
+} from "../../constants/colors";
+
+type Props = {|
+  onSelectedIndexChange: number => void,
+  selectedIndex: ?number,
```

Comment:
Can we follow the same API as the `Picker` component and use `selectedValue` and `onValueChange` instead? We could also add a SegmentItem component to follow the same child based API too, especially if we need to add accessibility labels for each item.

Reply:
Personally I prefer the strategy of defining values/options in JSX. Especially when you restrict the children type to a particular class. I'll let you make the call.

Reply:
I changed the API to look the same as the `Picker` and I actually quite like it.